joern-cli/frontends/csharpsrc2cpg/src/test/scala/io/joern/csharpsrc2cpg/querying/dataflow/SimpleDataflowTests.scala

```diff
@@ -0,0 +1,26 @@
+package io.joern.csharpsrc2cpg.querying.dataflow
+
+import io.joern.csharpsrc2cpg.testfixtures.CSharpCode2CpgFixture
+import io.shiftleft.semanticcpg.language.*
+import io.joern.dataflowengineoss.language.*
+
+class SimpleDataflowTests extends CSharpCode2CpgFixture(withDataFlow = true) {
+
+  "a source with a simple re-assignment" should {
+
+    val cpg = code(basicBoilerplate("""
+        |int i = 0;
+        |int a = i + 1;
+        |Console.WriteLine(a);
+        |""".stripMargin))
+
+    "still propagate to the sink" in {
+      cpg.method("Main").dotAst.foreach(println)
+      val src  = cpg.assignment.target.isIdentifier.nameExact("i").l
+      val sink = cpg.call.nameExact("WriteLine").l
+      sink.reachableBy(src).p.foreach(println)
```

Comment:
Is this a debug statement? We might want to remove this.

Reply:
Ah yes good catch